### DISCUSSION TITLE
Problem with open an .jpg image shot by a canon

### DISCUSSION THREAD

**Discussion by Atomzwieback** [davidbyttow/govips#439] [Q&A]
Hi,

i stumbled over a problem that vips is not able to understand the image format. All those pictures i have the issue with are a .jpg and shot by a "Canon EOS 6D Mark II". Vips throws the error "govips.info] failed to understand image format size=431". Any hint why this could be a problem?

**Comment by tonimelisma** [comment]
Hey! Thanks for the comment. I'm travelling overseas for a few weeks and generally we don't have resources to help debug individual errors as there's loads of different image formats and libraries here that could be the cause. I would try to open the image with the libvips CLI directly, check compiled dependencies, etc. The project is open source.

Apologies I can't be of more help.
